core/src/main/resources/lib/layout/tab.jelly

```diff
@@ -22,24 +22,32 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
-<!--
-<%@ attribute name="name" required="true" type="java.lang.String" %>
-<%@ attribute name="href" required="true" type="java.lang.String" %>
-<%@ attribute name="active" required="true" type="java.lang.Boolean" %>
-<%@ attribute name="title" required="false" type="java.lang.String" %>
--->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core">
-  <div class="tab${active?' active':''}">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+  <st:documentation>
+    <st:attribute name="name" use="required">
+      The name of the tab
+    </st:attribute>
+    <st:attribute name="href" use="required">
+      The url of the tab
+    </st:attribute>
+    <st:attribute name="active" type="java.lang.Boolean">
+      Whether the tab is active or not
+    </st:attribute>
+    <st:attribute name="title">
+      The title of the tab
```

Comment:
I am restoring behaviour that was broken in https://github.com/stephenc/jenkins/commit/2de93f861a8e6997d8f4b7b186b7953c835771cc#diff-1948650f90f2cefd48aff59fa5742aa0L58 (appears to be removed by omission as https://github.com/stephenc/jenkins/blob/2de93f861a8e6997d8f4b7b186b7953c835771cc/core/src/main/resources/lib/layout/tab.jelly#L29 was retained